.env

```diff
@@ -3,4 +3,11 @@ PGDATABASE='TimeOffDB'
 PGUSER='TimeOffDB_owner'
 PGPASSWORD='wKT0JAWF6BqD'
 ENDPOINT_ID='ep-empty-voice-a2dpidxi'
-PORT=4050
\ No newline at end of file
+PORT=4051
```

Comment:
Is there a specific reason for deciding to change the port?

Reply:
We had issues with ports as it always complained about port already in use. Now backend is on 4051 and front on 3000 and its finally resolved.